tests/foreman/api/test_rhci.py

```diff
@@ -0,0 +1,125 @@
+"""Unit tests for the ``fusor/api/v21/deployments`` paths."""
+from ddt import data, ddt
+from fauxfactory import gen_string
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo.test import APITestCase
+# (too-many-public-methods) pylint:disable=too-many-public-methods
+
+
+@ddt
+class RHCIDeploymentTestCase(APITestCase):
+    """Tests for the RHCI deployment API."""
+
+    @classmethod
+    def setUpClass(cls):
+        """ Setup the base entities for all tests. """
+        super(RHCIDeploymentTestCase, cls).setUpClass()
+        cls.org = entities.Organization().create_json()
```

Comment:
You're having an issue with the RHCI API producing undesirable responses, right? How about this:

```python
org_id = entities.Organization().create_json()['organization']['id']
cls.org = entities.Organization(id=org_id)
```

You now have a valid Organization entity.